tools/nightly.py

```diff
@@ -212,6 +213,28 @@ def check_branch(subcommand: str, branch: str | None) -> str | None:
     return None
 
 
+def check_conda_env_exists(name: str | None = None, prefix: str | None = None) -> bool:
+    """Checks that the conda environment exists."""
+    if name is not None and prefix is not None:
+        raise ValueError("Cannot specify both --name and --prefix")
+    if name is None and prefix is None:
+        raise ValueError("Must specify either --name or --prefix")
+
+    try:
+        cmd = ["conda", "info", "--envs"]
```

Comment:
nit use `--json` here for programmatic access

Reply:
`conda info --envs --json` only shows the `prefix` path to environments. The `name` of the environments are not returned.